src/assets/style-components/header/_themeToggle.scss

```diff
@@ -0,0 +1,65 @@
+@use '../mixins' as *;
+@use '../variables' as *;
+
+//Import icons for theme toogle button
+@import url(https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css);
```

Comment:
As we have the `react-icons` package now, we should remove this and use those instead

Reply:
We’re not using `react-icons` package for ‘themeToggle’ so not necessarily to remove it.

Reply:
Change it using Component approach of react icons and styled them inline.